pyproject.toml

```diff
@@ -111,7 +111,7 @@ omit = ["betterproto/tests/*"]
 legacy_tox_ini = """
 [tox]
 isolated_build = true
-envlist = py36, py37, py38
+envlist = py36, py37, py38, py310
```

Comment:
I noticed you're already running 310 in CI so figured it wouldn't hurt to add to tox, which I then used to test on 3.6 and 3.10